proof/crefine/AARCH64/Syscall_C.thy

```diff
@@ -2229,26 +2229,42 @@ lemma ccorres_handleReservedIRQ:
                   simp flip: word_unat.Rep_inject split: if_splits)
   done
 
+(* Because of a case distinction, this proof occurs multiple times in handleInterrupt_ccorres below.
+   The proof is carefully manual to match up the C expression "if (!defined(CONFIG_ARM_GIC_V3))",
```

Comment:
again, I'm not sure what `defined()` is, did you mean `config_set()` or am I blind